doc/faq.rst

```diff
@@ -3,6 +3,25 @@ FAQ
 
 This is the list of Frequently Asked Questions for restic.
 
+Can restic resume an interrupted backup?
+----------------------------------------
+
+Yes it can. When backing up, restic periodically writes index files to keep a record of the uploaded data.
+Even if there's no snapshot created in the end (due to the backup being interrupted), these indexes are
+stored in the repository for the data that has been uploaded so far. Next time restic runs, it is then able
+to find the uploaded data through these indexes, and thereby reference it again without having to upload
+it a second time. This effectively makes it continue from where it saved the last index, which is recently.
```

Comment:
I'd remove "which is recently".

Reply:
If we remove this, it becomes less clear to the user/reader that it's not a long time and thereby won't be a lot of data that's not re-used, I think. At first I wrote "up to 10 minutes", but don't want to put fixed numbers in the documentation. At this point there's nothing in the text that explains how regularly the indexes are created, and with that I feel there is room for them thinking it might be an hour or whatever, it's really undefined. Thoughts?

Reply:
Changed the text, see what you think.

Reply:
How about replacing "periodically" (first sentence) by "every 10 minutes" and adding a comment in the code (# promised in the FAQ)?

Reply:
The word "periodically" was chosen because it's what fd0 used in the PR adding this functionality to restic, and it's a perfectly valid word. Could be replaced with "regularly", doesn't matter to me.

Replacing it with a fixed number is not good because it adds another data point we have to maintain in case those 10 minutes ever change. We should try to keep documentation as neutral and maintenance-free as possible.

Reply:
As a user, I'd like to have a definite amount of time to wait. As a regular contributor of patches, I'd heed a comment above `repository.indexMagAge = 10 * time.Minute` telling me to `Update the FAQ before changing this constant`. That's all I was trying to say.

Reply:
Yeah, I see. But it makes for a mess. Even if contributors see that, it's still a mess. I'm sorry but it should be enough with what it says now, since "a few minutes" isn't expected to mean something like half an hour or more. But if it's better to remove it that's fine too. I don't want to depend on cross-referencing comments in code though.

Reply:
@greatroar I will go ahead and merge it as it is after having discussed the wording with @MichaelEischer as well. Thanks for your input though, it's always appreciated even if we don't agree fully on specifics all the time :)